diff --git a/addons/oca/mail/mail_tracking/models/mail_mail.py b/addons/oca/mail/mail_tracking/models/mail_mail.py
index 74e1c7f..d1f490a 100644
--- a/addons/oca/mail/mail_tracking/models/mail_mail.py
+++ b/addons/oca/mail/mail_tracking/models/mail_mail.py
@@ -29,7 +29,7 @@ class MailMail(models.Model):
         }
 
     def _prepare_outgoing_list(
-        self, mail_server=False, recipients_follower_status=None
+        self, mail_server=False, recipients_follower_status=None, **kwargs
     ):
         """Creates the mail.tracking.email record and adds the image tracking
         to the email. Please note that because we can't add mail headers in this
@@ -37,7 +37,7 @@ class MailMail(models.Model):
         also be used to extract the mail.tracking.email record id and to set the
         X-Odoo-MailTracking-ID header there.
         """
-        emails = super()._prepare_outgoing_list(mail_server, recipients_follower_status)
+        emails = super()._prepare_outgoing_list(mail_server=mail_server, **kwargs)
         tracking_vals_list = [self._tracking_email_prepare(email) for email in emails]
         trackings = self.env["mail.tracking.email"].sudo().create(tracking_vals_list)
         for email, tracking in zip(emails, trackings, strict=True):
